sbt_common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Work.scala

```diff
@@ -40,48 +40,48 @@ trait Work extends BaseWork with MultipleSourceIdentifiers {
-  otherIdentifiers: List[SourceIdentifier] = List(),
-  mergeCandidates: List[MergeCandidate] = List(),
-  workType: Option[WorkType] = None,
-  description: Option[String] = None,
-  physicalDescription: Option[String] = None,
-  extent: Option[String] = None,
-  lettering: Option[String] = None,
-  createdDate: Option[Period] = None,
-  subjects: List[Subject[MaybeDisplayable[AbstractConcept]]] = Nil,
-  genres: List[Genre[MaybeDisplayable[AbstractConcept]]] = Nil,
-  contributors: List[Contributor[MaybeDisplayable[AbstractAgent]]] = Nil,
-  thumbnail: Option[Location] = None,
-  production: List[ProductionEvent[MaybeDisplayable[AbstractAgent]]] = Nil,
-  language: Option[Language] = None,
-  dimensions: Option[String] = None,
-  items: List[Identifiable[Item]] = Nil,
+  workType: Option[WorkType],
+  description: Option[String],
+  physicalDescription: Option[String],
+  extent: Option[String],
+  lettering: Option[String],
+  createdDate: Option[Period],
+  subjects: List[Subject[MaybeDisplayable[AbstractConcept]]],
+  genres: List[Genre[MaybeDisplayable[AbstractConcept]]],
+  contributors: List[Contributor[MaybeDisplayable[AbstractAgent]]],
+  thumbnail: Option[Location],
+  production: List[ProductionEvent[MaybeDisplayable[AbstractAgent]]],
+  language: Option[Language],
+  dimensions: Option[String],
+  items: List[Identifiable[Item]],
+  version: Int,
   ontologyType: String = "Work")
     extends Work
     with TransformedBaseWork
 
 case class IdentifiedWork(
   canonicalId: String,
   sourceIdentifier: SourceIdentifier,
-  otherIdentifiers: List[SourceIdentifier] = List(),
-  mergeCandidates: List[MergeCandidate] = List(),
+  otherIdentifiers: List[SourceIdentifier],
+  mergeCandidates: List[MergeCandidate],
   title: String,
-  workType: Option[WorkType] = None,
-  description: Option[String] = None,
-  physicalDescription: Option[String] = None,
-  extent: Option[String] = None,
-  lettering: Option[String] = None,
-  createdDate: Option[Period] = None,
-  subjects: List[Subject[Displayable[AbstractConcept]]] = Nil,
-  genres: List[Genre[Displayable[AbstractConcept]]] = Nil,
-  contributors: List[Contributor[Displayable[AbstractAgent]]] = Nil,
-  thumbnail: Option[Location] = None,
-  production: List[ProductionEvent[Displayable[AbstractAgent]]] = Nil,
-  language: Option[Language] = None,
-  dimensions: Option[String] = None,
-  items: List[Identified[Item]] = Nil,
+  workType: Option[WorkType],
+  description: Option[String],
+  physicalDescription: Option[String],
+  extent: Option[String],
+  lettering: Option[String],
+  createdDate: Option[Period],
+  subjects: List[Subject[Displayable[AbstractConcept]]],
+  genres: List[Genre[Displayable[AbstractConcept]]],
+  contributors: List[Contributor[Displayable[AbstractAgent]]],
+  thumbnail: Option[Location],
+  production: List[ProductionEvent[Displayable[AbstractAgent]]],
+  language: Option[Language],
+  dimensions: Option[String],
+  items: List[Identified[Item]],
```

Comment:
The change here is to reorder the fields on UnidentifiedWork and IdentifiedWork so they match, which should make for easier comparisons.